data.json

```
@@ -880,5 +880,11 @@
     "githubUsername": "wordssaysalot",
     "imageUrl": "https://user-images.githubusercontent.com/56651899/118150685-71773600-b430-11eb-95f7-38a7479c5e34.png",
     "issueNumber": 498
+  },
+  {
+    "name": "Ibrahim Ramadhan",
+    "githubUsername": "ibby360",
+    "imageUrl": "![github]https://user-images.githubusercontent.com/51038084/118176523-31b54c00-b43a-11eb-8e62-bcc439335b8f.png",
```

```suggestion
    "imageUrl": "https://user-images.githubusercontent.com/51038084/118176523-31b54c00-b43a-11eb-8e62-bcc439335b8f.png",
```

This md tag is not necessary here

---

Changes have been made as requested